apps/frontend/src/extra_components/Carousel.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { useEffect, useMemo, useState } from "react";
+import {
+  Box,
+  IconButton,
+  Card,
+  Slide,
+  Typography,
+  Grid,
+  Paper,
+} from "@mui/material";
+import ArrowLeftIcon from "@mui/icons-material/ArrowLeft";
+import ArrowRightIcon from "@mui/icons-material/ArrowRight";
+
+export interface props {
+  imageURLs: string[];
+}
+const Carousel = ({ imageURLs }: props) => {
+  const [cards, setCards] = useState<React.ReactElement[]>([]);
```

Comment:
ReactElement should be imported at the top of the file along with useEffect, useState, etc.

Reply:
Sounds good, will do!